### PR TITLE
PIMOB-1375: Fix iPad scroll view background

### PR DESCRIPTION
**Pull request by ehab-al-cko**
[Jira ticket ](https://checkout.atlassian.net/browse/PIMOB-1375)

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [ ] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [ ] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: iOS Example Frame/iOS Example Frame/Style/CustomStyle1.swift [hunk 1]
```diff
@@ -51,7 +51,7 @@ struct PayButtonFormStyleCustom1: ElementButtonStyle {
 // MARK: - Main Payment Form
 
 struct PaymentFormStyleCustom1: PaymentFormStyle {
-  var backgroundColor: UIColor = Constants.backgroundColor
+  var backgroundColor: UIColor = .white
```
Comment (alex-ioja-yang-cko) [inline iOS Example Frame/iOS Example Frame/Style/CustomStyle1.swift:54]: I am unsure if this is supposed to get committed? Makes sense you tested it, but CustomStyle1 is still matching a design

File: iOS Example Frame/iOS Example Frame/Factory/Factory.swift [hunk 2]
```diff
@@ -106,10 +106,15 @@ struct Factory {
...
 
-    let viewController = PaymentFormFactory.buildViewController(configuration: configuration,
-                                                                style: style,
+      let paymentFormStyle = Style.paymentForm
+
+      let style =  PaymentStyle(paymentFormStyle: paymentFormStyle,
+                                billingFormStyle: billingFormStyle)
+
+      let viewController = PaymentFormFactory.buildViewController(configuration: configuration,
+                                                                  style: style,
```
Comment (alex-ioja-yang-cko) [inline iOS Example Frame/iOS Example Frame/Factory/Factory.swift:117]: Please undo.

File: iOS Example Frame/iOS Example Frame/Style/CustomStyle1.swift (hunk 1 above)
Comment (ehab-al-cko) [reply iOS Example Frame/iOS Example Frame/Style/CustomStyle1.swift:54]: There's no custom style 1 design. other is a duplicate for Matrix style. It just a test ground.

File: iOS Example Frame/iOS Example Frame/Factory/Factory.swift (hunk 2 above)
Comment (alex-ioja-yang-cko) [reply iOS Example Frame/iOS Example Frame/Factory/Factory.swift:117]: This was theme 2 and if you look at line 109 it was using the ThemeDemo object to showcase the code required to use Theme approach :).

**Review by alex-ioja-yang-cko — APPROVED** [review]
Please pull latest commits from base branch to get pipeline green and merge

**Pull request merged**